packages/lit-dev-content/site/docs/localization/overview.md

```diff
@@ -203,9 +209,8 @@ their own advantages.
 
 <div class="alert alert-info">
```

Comment:
I know I recommended the run-in head, but looking at this section rendered on the web site, I realize that it looks kind of funny to have one sentence followed by a note. With both the initial version and my suggestion, it implies that the reader some kind of basis for deciding between the modes... But you haven't described them yet.

Often in this situation I use a bullet list with a brief summary of each option, then go into the full sections. Something like:

```
*   _runtime_ mode uses Lit Localize's APIs to load localized messages at runtime.
*   _transform_ mode eliminates the Lit Localize runtime code by building a separate JavaScript bundle for each locale. 
```

... Which could be followed either by the note (as currently formatted) or your original text, as a plain paragraph.

This just gives people a little preview of what they're going to find in the following sections.

Reply:
Done